src/vxingest/builder_common/vx_ingest.py

```diff
@@ -13,13 +13,13 @@
 threading model.
 """
 
+import datetime as dt
 import json
 import logging
 import os
+import pathlib
 import sys
 import time
-from datetime import timedelta
-from pathlib import Path
```

Comment:
Was there a reason to switch `from pathlib import Path` to `import pathlib`? The first is more in line with the rest of the codebase. 

Likewise we could `from datetime import timedelta, strptime` instead of importing the whole of `datetime`.